src/TuDa.CIMS.Web/Components/AssetSearch.razor

```diff
@@ -0,0 +1,22 @@
+﻿@* Search bar for assets *@
+<div class="position-relative"
+     style="max-width: 600px;">
+    <input class="form-control rounded-pill"
+           id="prototypeSearchBar"
+           placeholder="Suche">
+
+    <!-- Search button -->
+    <button type="button"
+            class="position-absolute end-0 top-50 translate-middle-y me-2 btn p-0 border-0 bg-transparent">
+
+        <!-- Search icon -->
+        <i class="bi bi-search position-absolute"
+           style="top: 50%; right: 15px; transform: translateY(-50%); color: #aaa;">
```

Comment:
:wrench: Can we use bootstrap classes for that, or extract this to `AssetSearch.razor.css` as a new class?